app/models/slot_range.rb

```diff
@@ -2,4 +2,8 @@ class SlotRange < ApplicationRecord
   belongs_to :user
   has_many :slots, inverse_of: :slot_range, dependent: :destroy
   accepts_nested_attributes_for :slots, allow_destroy: true
+
+  def display_title
+    "#{from.strftime('%d %B %Y')} ➡ onwards"
```

Comment:
Don't think we need that quite yet because this UI could change a bit... Later on I was going to change onwards to display "whatever the next slot range from is"... except for the last one.